unit_test/include/one_qubit_register_test.hpp

```diff
@@ -37,6 +37,20 @@ class OneQubitRegisterTest : public ::testing::Test
 //////////////////////////////////////////////////////////////////////////////
 // Test macros:
 
+TEST_F(OneQubitRegisterTest, InitializeWithDefault)
```

Comment:
As mentioned in another comment, this unit test is run only when there are at most 1 MPI ranks.